autotest/gcore/mask.py

```diff
@@ -1151,25 +1151,12 @@ def mask_26():
     if ds.GetRasterBand(1).GetMaskFlags() != gdal.GMF_ALPHA + gdal.GMF_PER_DATASET:
         gdaltest.post_reason('Did not get expected mask.')
         return 'fail'
-    mask = ds.GetRasterBand(1).GetMaskBand()
-
-    # IRasterIO() optimized case
```

Comment:
ah, this case  "#IRasterIO() optimized case" was the interesting one to let. The "special case where dynamics is only 0-255" is not so interesting